### PR TITLE
Small jungle trees

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
Ausops sprited them they're good I promise

### CODE REVIEW

**Comment by Armhulen** [comment]
SPRIIIIIIIIIIIIIIIIIIIIIIITES!

**Comment by Thunder12345** [comment]
Pictures please

**Comment by KorPhaeron** [comment]
Scared?

**Comment by WJohn** [comment]
pretty!

**Comment by Armhulen** [comment]
ok good!

**Comment by Cyberboss** [comment]
Not sure if @MrStonedOne meming or the tagger bot going rogue...

File: code/game/objects/structures/flora.dm [hunk 1]
```diff
@@ -91,6 +91,10 @@
 	icon_state = "[icon_state][rand(1, 6)]"
 	..()
 
+/obj/structure/flora/tree/jungle/small
+	pixel_y = 0
+	pixel_x = -32
```
Comment (Cyberboss) [inline code/game/objects/structures/flora.dm]: This doesn't change the icon or anything?

**Comment by FantasticFwoosh** [comment]
This is planetstations own aesthetic, but to our current code & sprites it sticks out like a thumb.

File: code/game/objects/structures/flora.dm (hunk 1 above)
Comment (Militaires) [inline code/game/objects/structures/flora.dm]: p sure it just offsets the sprite cause it's so giant

**Comment by PraiseRatvar** [comment]
tree**s** THERE ARE MULTIPLE, I WANT TO SEE ALL THE TREES

**Comment by MrStonedOne** [comment]
@Cyberboss I'm not even near a computer that has the password store for tg accounts like the github bot

**Comment by DoctoreMad** [comment]
They are pretty good form the initial image, yes.

**Comment by Lumipharon** [comment]
Neat.

File: code/game/objects/structures/flora.dm (hunk 1 above)
Comment (RemieRichards) [inline code/game/objects/structures/flora.dm]: no, he's commenting that there' no ```icon_state = ...``` line, meaning the small tree will look like a big tree

Comment (AnturK) [inline code/game/objects/structures/flora.dm]: @KorPhaeron cmon.

Comment (Cyberboss) [inline code/game/objects/structures/flora.dm]: FIX IT

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/flora.dm (hunk 1 above)
Comment (KorPhaeron) [inline code/game/objects/structures/flora.dm]: I fixed it

**Review by Cyberboss — APPROVED** [review]

File: code/game/objects/structures/flora.dm [hunk 2]
```diff
@@ -382,4 +387,4 @@
 
 /obj/structure/flora/rock/pile/largejungle/Initialize()
 	..()
```
Comment (Cyberboss) [inline code/game/objects/structures/flora.dm]: . = ..() while you're here actually

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**